src/data/print_documents/documents.py

```diff
@@ -404,6 +415,33 @@ def static_name() -> str:
     def static_id() -> str:
         return 'pairings'
 
+    @staticmethod
+    def available_options() -> list[type[PrintOption]]:
+        return [PairingStylePrintOption, RoundPrintOption]
+
+    @override
+    @property
+    def ordered_players(self) -> list[Player]:
+        assert self.tournament is not None
+        return self.tournament.players_by_name_without_unpaired
+
+    @override
+    @property
+    def template_name(self) -> str:
+        return self._get_option(PairingStylePrintOption).pairing_style.template
+
+    @override
+    @property
+    def is_pairings_list(self) -> bool:
+        return True
+
+    @property
+    def template_context(self) -> dict[str, Any]:
+        context = {}
+        context.update(BoardPrintDocument.template_context.fget(self))
+        context.update(PlayerPrintDocument.template_context.fget(self))
+        return context
```

Comment:
```suggestion
        return (
            BoardPrintDocument.template_context.fget(self) | 
            PlayerPrintDocument.template_context.fget(self)
        )
```
merge dict operator `|` was introduced to improve `.update()`'s lack of readability, its usage is prefered